app/src/main/java/com/example/admin/learningandroid/layouts/ModelLoginPage.java

```diff
@@ -0,0 +1,49 @@
+package com.example.admin.learningandroid.layouts;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.example.admin.learningandroid.R;
+
+public class ModelLoginPage extends AppCompatActivity implements View.OnClickListener {
+    public EditText etName;
+    public EditText etMarks;
+    public EditText etRegistrationNumber;
+    public Button btnSave;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.model_login_page);
+        etName = findViewById(R.id.et_name);
+        etMarks = findViewById(R.id.et_marks);
+        etRegistrationNumber = findViewById(R.id.et_registration);
+        btnSave = findViewById(R.id.btn_save);
+        btnSave.setOnClickListener(this);
+    }
+
+    @Override
+    public void onClick(View view) {
+        String mName = etName.getText().toString();
+        String mMarks = etMarks.getText().toString();
+        String mRegistrationNumber = etRegistrationNumber.getText().toString();
+        int mCount = mName.length();
+        if (mCount <= 3) {
```

Comment:
you can opt for early return.. 

For eg : 
if(a>b){
// do something ;
return;
}
if (c>d){
// do something return 
}